### PR TITLE
[JENKINS-28649] Api call with AfterRestartTask throws ExportedBean annotation exception

### PR DESCRIPTION
**Pull request by christ66**
- [ ] functional test using HtmlUnit to retrieve (e.g.) `api/json`
- [X] `AfterRestartTask`
- [ ] `PlaceholderTask`

@reviewbybees

### CODE REVIEW

File: job/src/main/java/org/jenkinsci/plugins/workflow/job/AfterRestartTask.java [hunk 1]
```diff
@@ -33,11 +33,13 @@
...
+import org.kohsuke.stapler.export.ExportedBean;
 
 /**
  * Represents a {@link WorkflowRun} still running after a Jenkins restart.
  * Could be a {@code ContinuedTask}, though not really necessary since it is a {@link FlyweightTask} which would never be blocked anyway.
  */
+@ExportedBean
 class AfterRestartTask extends AbstractQueueTask implements Queue.FlyweightTask, Queue.TransientTask {
 
     private final WorkflowRun run;
```
Comment (jglick) [inline job/src/main/java/org/jenkinsci/plugins/workflow/job/AfterRestartTask.java]: This is what should be `@Exported` I think; this class is just a proxy for the run.

Comment (christ66) [inline job/src/main/java/org/jenkinsci/plugins/workflow/job/AfterRestartTask.java]: Fixed.

**Comment by jglick** [comment]
Would be appropriate to have test coverage for this, but did you at least test this (as of the second commit) interactively to verify that (a) no exception is thrown, (b) usable information is available from `/queue/api/xml`?

**Comment by kaihowl** [comment]
This seems to be possible fix for JENKINS-28649 (critical). What work is left to merge this in? We would very much like to be able to see the build queue in the REST API again. Thank you for working on this. :)

**Comment by amuniz** [comment]
This PR has been completed in #298 

@christ66 your commits have been included in the other PR, could you close this one please?

**Comment by jglick** [comment]
@amuniz there is no need to close this one. If and when #298 is merged, this one will be shown as merged automatically as well: its latest commit will be an ancestor of the target branch.

**Pull request merged**